change/@microsoft-teams-js-61f74ab4-102e-4a46-88bb-55dbf245fbc1.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add an additional parameter for Handoff ID in `composeMailWithHandoffId` API ",
```

```suggestion
  "comment": "Added the`mail.handoff.composeMailWithHandoffId` function that allows passing a handoff id to the compose mail window.",
```

You may want to check with PM to find out what specific language to write here since this string will show up in the patch notes for developers.